SUMMARY.md

```diff
@@ -0,0 +1,40 @@
+## Table of Contents
+
+* [Classes](content/Classes/Classes.q.md) ** -  minutes**
+    > Classes are blueprints for objects. In this lesson, you will learn the basic building blocks of a class, its capabilities, and its uses.  
+
+* [Objects](content/Objects/Objects.q.md) ** -  minutes**
+    > Objects are the actual representation of a class. Discover how you can create objects from a class using the `new` keyword.
+
+* [Properties](content/Properties/Properties.q.md) ** -  minutes**
+    > Properties are the variables defined inside a class. In this lesson, you will learn how to add properties to a class. 
+
+* [Methods](content/Methods/Methods.q.md) ** -  minutes**
+    > Methods are one of the basic building blocks of classes. In this lesson, you will learn how to add methods to a class. 
+
+* [Visibility](content/Visibility/Visibility.q.md) ** -  minutes**
+    > Explore the things you can do to manipulate the methods and properties of a class. In this lesson, you will learn what visibility is, how it works, and why it is important in Object Oriented PHP.
+
+* [Constructor](content/Constructor/Constructor.q.md) ** -  minutes**
+    > Learn how you can use the `__construct()` method to define a constructor for a class.
+
+* [Inheritance](content/Inheritance/Inheritance.q.md) ** -  minutes**
+    > Discover how classes inherit the methods and properties of another class using the `extends` keyword.
+
+* [Abstract Classes and Methods](content/AbstractClassesAndMethods/AbstractClassesAndMethods.q.md) ** -  minutes**
+    > Learn how you can use the `abstract` keyword to declare classes and methods as `abstract`.
+
+* [Interfaces](content/Interfaces/Interfaces.q.md) ** -  minutes**
+    > Interfaces are outlines of what an object can do. In this lesson, you will learn how to work with `interfaces` to define `public methods` that a class must `implement`.
+
+* [Static Methods and Properties](content/StaticMethodsAndProperties/StaticMethodsAndProperties.q.md) ** -  minutes**
+    > Discover how you can make class properties and methods accessible without an instance of the created object. 
+
+* [Constants](content/Constants/Constants.q.md) ** -  minutes**
+    > In this lesson, you will learn how to declare and access constants within classes.
+
+* [Autoloading Classes](content/AutoloadingClasses/AutoloadingClasses.q.md) ** -  minutes**
+    > Discover how you can organize and load class files automatically using `spl_autoload_register()` and `__autoload()` functions.
+
+* [Namespaces](content/Namespaces/Namespaces.q.md) ** -  minutes**
+    > Discover how you can group related classes, interfaces, methods and constants using the `namespace` keyword.
```

Comment:
Thank you for the suggestions. I think this is not necessary to always start our statements with a verb since if we have try to revisit all our existing courses outline we did not starts all our statements with verbs. Like this one also in most of my references:
![image](https://user-images.githubusercontent.com/25945790/58867542-39680400-86ed-11e9-94fb-0dd887c95135.png)

![image](https://user-images.githubusercontent.com/25945790/58867628-5e5c7700-86ed-11e9-8f92-37a7b8b1ceb7.png)

![image](https://user-images.githubusercontent.com/25945790/58867684-78965500-86ed-11e9-83bd-dc96f3f2b86e.png)

Reply:
Hope this helps. Thanks.